src/main/java/com/github/juliarn/npc/NPC.java

```diff
@@ -290,6 +291,31 @@ public LabyModModifier labymod() {
     return new LabyModModifier(this);
   }
 
+  /**
+   * Creates a new teleport modifier which serves methods to teleport an NPC.
+   *
+   * @return a teleport modifier modifying this NPC
+   * @since 2.7-SNAPSHOT
+   */
+  @NotNull
+  public TeleportModifier teleport() {
+    return new TeleportModifier(this);
+  }
+
+  /**
+   * Method used to update only the variable of the position.
+   *
+   * @param location new location of npc.
+   * @deprecated No supported for public use, changing the location of the npc would only lead to problems.
+   * If you want to teleport the NPC instead you can use {@link #teleport()}.
+   * @since 2.7-SNAPSHOT
+   */
+  @ApiStatus.Internal
+  @Deprecated
```

Comment:
I think we can remove the deprecated thing here, just mark it as internal and move the deprecation notice in the javadoc behind the actual description 🤔